defaults/main.yml

```diff
@@ -12,3 +12,14 @@ cmd_openstack_release: pike
 
 # whether to enable or not the candidate repo
 cmd_enable_candidate_repo: false
+
+# boolean to execute tasks on igtf.yml
```

Comment:
Please update documentation, it's not clear why it's name `ca_verification` and what is it for.

Reply:
:+1:  it's done in the last amended push